core/src/executor/fetch.rs

```diff
@@ -260,7 +260,10 @@ pub async fn fetch_relation_rows<'a, T: GStore>(
                         let rows = schemas.into_iter().map(move |schema| {
                             Ok(Row::Vec {
                                 columns: Rc::clone(&columns),
-                                values: vec![Value::Str(schema.table_name)],
+                                values: vec![
+                                    Value::Str(schema.table_name),
+                                    Value::Str(schema.comment.unwrap_or_default()),
+                                ],
```

Comment:
currently user may be confused with NULL and `""` .

gluesql> create table x (id int) comment = 'a';
Table created

gluesql> create table y (id int);
Table created

gluesql> select * from glue_tables;
| TABLE_NAME | COMMENT |
|------------|---------|
| x          | a       |
| y          |         |

gluesql> select * from glue_tables where COMMENT is null;
| TABLE_NAME | COMMENT |
|------------|---------|

If we direclty fetch NULL, it would be more clear

```rust
let rows = schemas.into_iter().map(move |schema| {
    let comment = match schema.comment {
      Some(comment) => Value::Str(comment),
      None => Value::Null,
    };
    
    Ok(Row::Vec {
      columns: Rc::clone(&columns),
      values: vec![
          Value::Str(schema.table_name),
          comment
      ],
    })
```

Reply:
We missed this point in #1469 as well.. 😢

Reply:
oh thanks! for now, I updated GLUE_TABLES and I will also work on GLUE_TABLE_COLUMNS in another PR. it would be also good to apply same stuff to DEFAULT.